site/docs/v1/tech/shared/_hosted-backend-response-cookies.adoc

```diff
@@ -0,0 +1,13 @@
+[.api]
+[field]#app.at# [type]#[String]#::
+The encoded access token. This cookie is written in the response as an HTTP Only session cookie.
+
+[field]#app.rt# [type]#[String]#::
+The refresh token. This cookie is written in the response as an HTTP only persistent cookie. The cookie expiration is configured in the JWT
+configuration for the application or the global JWT configuration.
+
+[field]#app.idt# [type]#[String]#::
+The Id token for the user for the configured application. Only present if the openid scope is requested. This is a JWT and can be accessed by JavaScript to display user account information.
```

Comment:
Please specify what kind of cookie this is as the second sentence, to keep it consistent with the previous two descriptions.